### ISSUE TITLE
Import in Eclipse (as maven project ?)

### ISSUE THREAD

**Issue by vdemeester** [domeide/doclipser#14]
What is (going) to be the best way to import the project in Eclipse to start contributing on it ? I would guess/recommend it `:P`.
- I imported it as Maven Project, but it modified the `.classpath` and `.project` files. From what I've seen, there is no absolute path in there so it should work if shared. I'll make a pull request with the change if we want to use that.
- We should create some predefined shared launch script so that it is really easy to get working on first contributions `:)`.

**Comment by l0rd** [comment]
**.classpath and .project files**
I would prefer not to add .project and .classpath. I did it because maven tycho build was failing on circleCI server. In particular com.zenika.docker.dsl.tests was failing and I thought adding .project and .classpath would fix that (it did it). But I will see if there is not a better way that including .project and .classpath files before merging the pull request.

**doc for contributing**
In the meantime I will document the way to setup the development environment. 

**launch scripts**
I thought I already share them but I did not. I need to remove *.launch from .gitignore file and commit them.
